security/plaintext-ldaps-auth-control-center/README.rst

```diff
@@ -0,0 +1,273 @@
+Deploy Confluent Platform
+=========================
+
+In this workflow scenario, you'll set up Control Center with LDAPs (ldap over TLS) based authentication to monitor and connect to a Confluent Platform with no security.
+
+The goal for this scenario is for you to:
+
+* Configure LDAP with TLS server for Control Center authentication (no RBAC).
+* Quickly set up the complete Confluent Platform on the Kubernetes.
+* Configure a producer to generate sample data.
+
+
+To complete this scenario, you'll follow these steps:
+
+#. Set the current tutorial directory.
+
+#. Deploy Confluent For Kubernetes.
+
+#. Deploy LDAPS server
+
+#. Deploy Confluent Platform.
+
+#. Deploy the Producer application.
+
+#. Tear down Confluent Platform.
+
+==================================
+Set the current tutorial directory
+==================================
+
+Set the tutorial directory for this tutorial under the directory you downloaded
+the tutorial files:
+
+::
+   
+  export TUTORIAL_HOME=<Tutorial directory>/plaintext-ldaps-auth-control-Center
+
+===============================
+Deploy Confluent for Kubernetes
+===============================
+
+#. Set up the Helm Chart:
+
+   ::
+
+     helm repo add confluentinc https://packages.confluent.io/helm
+
+
+#. Install Confluent For Kubernetes using Helm:
+
+   ::
+
+     helm upgrade --install operator confluentinc/confluent-for-kubernetes --namespace=confluent
+  
+#. Check that the Confluent For Kubernetes pod comes up and is running:
+
+   ::
+     
+     kubectl get pods --namespace=confluent
+
+===============
+Deploy OpenLDAP
+===============
+
+This repo includes a Helm chart for `OpenLdap
+<https://github.com/osixia/docker-openldap>`__. The chart ``values.yaml``
+includes the set of principal definitions that Confluent Platform needs for
+RBAC.
+
+#. Deploy OpenLdap
+
+   ::
+
+     helm upgrade --install -f $TUTORIAL_HOME/openldapssl/ldaps.yaml test-ldap $TUTORIAL_HOME/../../assets/openldap --namespace confluent
```

Comment:
may need to do a `--set tls.enabled=true` here if you cant change it above

Reply:
since there is no impact I suspect leaving it as a true in the values should be the default, for future workflows as well.